### ISSUE TITLE
Add classical architectures

### ISSUE THREAD

**Issue by omarsar** [dair-ai/ml-visuals#14]
Some ideas for figures to add to the PPT

- [ ] Linear regression, single-layer neural network
- [ ] Multilayer Perceptron with hidden layer
- [ ] Backpropagation illustration
- [ ] Computational Graphs
- [ ] Dropout illustration
- [ ] CNN - Padding stride, pooling, 
- [ ] LeNet
- [ ] AlexNet
- [ ] VGG
- [ ] NiN 
- [ ] GoogleNet
- [ ] ResNet
- [ ] DenseNet
- [ ] RNN
- [ ] Deep RNNs
- [ ] Bidirectional RNNs
- [ ] GRU
- [ ] LSTM
- [ ] Language RNN models
- [ ] Backpropagation through time
- [ ] Encoder-Decoder Architecture
- [ ] Sequence to sequence with RNN encoder-decoder
- [ ] Bearm Search and alternatives
- [ ] Attention
- [ ] Multi-head attention
- [ ] Self-attention
- [ ] Transformer
- [ ] Word2vec/GloVe/BERT/....
- [ ] Common/Popular CV/NLP Tasks

**Comment by elie198** [comment]
U

**Comment by heany** [comment]
Maybe, the generative adversarial networks and graph neural networks should be included in the list.

**Comment by narutomst** [comment]
Deep Belief Networks

**Comment by nashid** [comment]
Can we Graph Neural Network models like GCN, GAT, GGNN, GraphSage, etc?

**Comment by nashid** [comment]
Contrastive Representation Learning models is also worth adding.
